docs/develop/03_workshop/02_setup.md

```diff
@@ -3,8 +3,6 @@ id: setup
 title: 🎒 Setup
 ---
 
-import Tabs from '@theme/Tabs';
```

Comment:
Same as above. We use it still in this file, but is not imported anymore? curious...

Reply:
https://github.com/KILTprotocol/docs/pull/147/commits/7aef98cc92d0acdc1243d70fc8db956147bdf67c